### PR TITLE
feat(main_menu): start with save slot page last modified

### PR DESCRIPTION
**Pull request by ThmsKnz**
This is a small enhancement for user experience when having multiple save games across many save game pages.

The current behavior is that when starting the game always the first save game page is shown containing save games 0 - 11. When a player has multiple save games he/she first needs to flip through the pages to the find the last modified save game.

This enhancement to the main menu sets the initial save slot page to the one containing the latest save game.

### CODE REVIEW

**Comment by ThmsKnz** [comment]
info: I've updated the commit as there were superfluous parantheses in the initial version.

File: addons/egoventure/nodes/main_menu.gd [hunk 1]
```diff
@@ -35,7 +35,7 @@ var _selected_slot: int
 var _is_save_mode: bool
 
 # The currently visible save slot page
-var _save_slot_page: int = 1
+var _save_slot_page: int = _get_save_slot_page_last_modified()
```
Comment (dploeger) [inline addons/egoventure/nodes/main_menu.gd]: I think, this call should be better placed at the end of the `configure` function.

**Review by dploeger — CHANGES_REQUESTED** [review]
Awesome! Thanks a lot. Just a small remark.

**Comment by ThmsKnz** [comment]
Thanks for the feedback. Done.

**Review by dploeger — APPROVED** [review]
Great work. Thanks a lot. 🎁

**Comment by dploeger** [comment]
@ThmsKnz Can you please always merge from develop? I've missed that on this one and accidently merged to main. No problem, just an info.

**Comment by ThmsKnz** [comment]
@dploeger Sorry about that, I must've accidentally checked out main before the push, will double check next time.

**Pull request merged**